source/index/contracts/Index.sol

```diff
@@ -148,13 +148,18 @@ contract Index is Ownable {
     * @dev start value of 0x0 starts at the head
     * @param start address Identifier to start with
     * @param count uint256 Number of locators to return
-    * @return bytes32[] result List of locators
+    * @return bytes32[] List of locators
+    * @return uint256[] List of scores corresponding to locators
+    * @return address The next identifier to provide for pagination
     */
   function getLocators(
     address start,
     uint256 count
-  ) external view returns (bytes32[] memory result) {
-
+  ) external view returns (
+    bytes32[] memory locators,
+    uint256[] memory scores,
+    address nextIdentifier
```

Comment:
should nextIdentifier be the name of the return term in line 189?

Reply:
I thought `nextIdentifier` was more descriptive of what was being returned to the called than `identifier`.

I could call it `nextIdentifier` throughout the function though

Reply:
yes, and i think that's accurate in terms how that variable is called. the initialized value is `entries[HEAD].next;`